strawberryfields/apps/qchem/dynamics.py

```diff
@@ -250,3 +254,62 @@ def prob(samples: list, excited_state: list) -> float:
         raise ValueError("The excited state must not contain negative values")
 
     return samples.count(excited_state) / len(samples)
+
+
+def marginal(mu: np.ndarray, V: np.ndarray, n: int, hbar: float = 2.0) -> np.ndarray:
+    r"""Generate marginal distributions from the displacement vector and covariance matrix of a
```

Comment:
```suggestion
    r"""Generate single-mode marginal distributions from the displacement vector and covariance matrix of a
```
It's important to clarify that these are the single-mode marginal distributions for each mode